### PR TITLE
[ISSUE #4721] remove code snippet It will never be executed

### LINKED ISSUE #4721: [Optimize]NamesrvStartup has some code snippet It will never be executed
as following:
image
NamesrvController.initialize never return false. this method return true or throw exception. so
```java
 if (!initResult) {
      controller.shutdown();
      System.exit(-3);
}
```
this code snippet can be removed.  
ControllerStartup class has the same problem

### PR DESCRIPTION
**Pull request by mxsm**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

Fix #4721 

## Brief changelog

- Remove NamesrvStartup some code snippet It will never be executed
- Remove ControllerStartup some code snippet It will never be executed

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by tsunghanjacktsai — COMMENTED** [review]
Guess these could be removed at the moment, but we should think about if it should always return true while initializing. @RongtongJin @hzh0425

File: controller/src/main/java/org/apache/rocketmq/controller/ControllerStartup.java [hunk 1]
```diff
@@ -135,11 +135,7 @@ public static ControllerManager start(final ControllerManager controller) throws
             throw new IllegalArgumentException("ControllerManager is null");
         }
 
-        boolean initResult = controller.initialize();
-        if (!initResult) {
-            controller.shutdown();
-            System.exit(-3);
-        }
+        controller.initialize();
```
Comment (lizhanhui) [inline controller/src/main/java/org/apache/rocketmq/controller/ControllerStartup.java:138]: If initialize always returns true, why not change it to void?

**Comment by mxsm** [comment]
I change it to void and the test class has also been modified. @lizhanhui @tsunghanjacktsai .

**Review by tsunghanjacktsai — APPROVED** [review]
Better now :-)

**Review by Oliverwqcwrw — COMMENTED** [review]
Hello @mxsm ,
Please resolve conflict

**Pull request closed without merging**